packages/aor-graphql-client/src/index.js

```diff
@@ -34,6 +34,7 @@ const getOptions = (options, aorFetchType, resource) => {
 
 export default async options => {
     const {
+        clientObject,
```

Comment:
I'd rather name this one `client` and rename the old `client` to `clientOptions`. We could then remove completely the check for ApolloClient at L48.

This will be a breaking change though and we need to update the documentation accordingly

Reply:
OK. I will change it. 
But what about tests?

If we need to chage tests I am going to take 1 or 2 days